redgenes/gene_annotations.py

```diff
@@ -343,6 +353,8 @@ def annotation_pipeline(df, tmpdir, kofamscan_profile, kofamscan_kolist):
             rrna_df.insert(0, "entity_id", entity_id[0])
             rrna_df["run_info"] = barrnap_run_id[0]
             args_rrna_info = rrna_df.values.tolist()
+            print("***** ready to insert with arguments")
+            print(args_rrna_info)
```

Comment:
```suggestion
```

library code usually doesn't have print statements as the output harder to manage. If this is useful long term, using the `logging` module would be preferrable